benchmarks/pca/objective.py

```diff
@@ -0,0 +1,66 @@
+from datetime import datetime
+
+from benchopt import BaseObjective
+
+
+class Objective(BaseObjective):
+    name = "PCA walltime"
+    url = "https://github.com/soda-inria/sklearn-engine-benchmarks"
+
+    requirements = ["numpy"]
+
+    # Since our goal is to measure walltime for solvers that perform exact same
+    # computations, the solver parameters are part of the objective and must be set
+    # for all solvers, rather than being an independent benchmark space for each
+    # solver.
+    parameters = dict(
+        n_components=[10],
+        whiten=[False],
+        tol=[0.0],
```

Comment:
Which solver requires a `tol` parameter?

Reply:
The 3 current solvers expose the parameters and it's only used by arpack and cupy's jacobi